tensorflow/python/keras/activations_test.py

```diff
@@ -31,10 +31,10 @@
 from tensorflow.python.platform import test
 
 
-def _ref_softmax(values):
-  m = np.max(values)
+def _ref_softmax(values, axis=-1):
```

Comment:
I think you can skip axis since its never changed by the caller.

Reply:
Sure let me modify it real quick.

Reply:
Done 😃